src/scripts/genEtwProvider.py

```diff
@@ -22,7 +22,12 @@
 etw_dirname = "etw"
 replacements = [
     (r"EventEnabled", "EventXplatEnabled"),
-    (r"\bPVOID\b", "void*")
+    (r"\bPVOID\b", "void*"),
+]
+counted_replacements = [
+    # This is a special case to fix https://github.com/dotnet/coreclr/issues/25324
+    # See https://github.com/dotnet/coreclr/pull/25454 for more information
```

Comment:
Can you add a concise version of the issue inline? Its a little awkward to ask people to go to a web page and read a lengthy discussion for an issue that can probably be summarized as:

"There is a bug in the MC code generator that miscomputes the size of ETW events which have variable size arrays of variable size arrays. This occurred in our GCBulkType event. This workaround replaces the bad size computation with the correct one. See `https://github.com/dotnet/coreclr/pull/25454` for more information"